konfera/event/views.py

```diff
@@ -127,3 +128,13 @@ def event_public_tickets(request, slug):
 
     context['tickets'] = available_tickets
     return render(request=request, template_name='konfera/event_public_tickets.html', context=context)
+
+
+def event_order_detail(request, slug, order_uuid):
+    context = dict()
+    event = get_object_or_404(Event.objects.published(), slug=slug)
```

Comment:
Okay, then `order.ticket_set.all()[0].type.event` could work. But it's really ugly so maybe it would be a good idea to add method `event` (with property decorator) which would return the event for the order.

We also have to add validation so an order can have multiple tickets only if they are for the same event. (I will create a new issue for that.)